.github/workflows/R-CMD-check.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
   R-CMD-check:
     runs-on: ${{ matrix.config.os }}
 
-    name: ${{ matrix.config.os }} (${{ matrix.config.r }}) ${{ matrix.config.locale }}
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
 
```

Comment:
We need to keep `${{ matrix.config.locale }}` here and several of the other changes below. Their purpose is to test vroom in a non-UTF-8 locale, which we should retain.